elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -79,11 +79,7 @@
   }
 }
 
-@mixin pfe-hidden {
```

Comment:
Did this get moved to somewhere else?

Reply:
inadvertent removal; it has been restored